submissions/madmaxWMFU/memory-game/style/style.css

```diff
@@ -0,0 +1,331 @@
+.main {
+    margin: 0 auto;
+    width: 50%;
+    text-align: center;
+}
+
+.game-caption {
+    font-weight: bold;
+    text-decoration: underline;
+    color: yellowgreen;
+}
+
+.game-field {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex-wrap: wrap;
+}
+
+.flipper {
+    transition: 0.6s;
+    transform-style: preserve-3d;
+    position: relative;
+    margin-bottom: 20px;
+}
+
+.flipper, .front, .back {
+    width: 159px;
+    height: 225px;
+}
+
+.front, .back {
+    border: solid yellowgreen;
+    border-radius: 20px;
+    cursor: pointer;
+	backface-visibility: hidden;
+	position: absolute;
+	top: 0;
+	left: 0;
+}
+
+.front {
+	z-index: 2;
+    transform: rotateY(0deg);    
+    background: url(../img/koloda.png) no-repeat -2228px -481px;
+    background-size: 2529px 947px;
+}
+
+.back {
+    transform: rotateY(180deg);
+    background: url('../img/koloda.png') no-repeat -2228px 0;
+    background-size: 2529px 947px;
+}
+
+.rotateCard {
+    transform: rotateY(180deg); 
+}
+
+.hideCard {    
+    transform: rotateY(0deg);
+}
+
+.card0 {
+    background: url('../img/koloda.png') no-repeat 0 0;
+    background-size: 2529px 947px;
```

Comment:
Can we define common properties for all cards in another css class ? `background-size` repeated.

Reply:
I rewrote the rule, but I can not remove the property `background-size`, without it the style does not work